config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -288,40 +289,38 @@ config MBEDTLS_ENABLE_HEAP
     default y
 
 config MBEDTLS_HEAP_SIZE
-    default 15360
-
-config MBEDTLS_TLS_LIBRARY
-    default y
+    default 15360 if !CHIP_CRYPTO_PSA
+    default 32768 if CHIP_CRYPTO_PSA
 
 config NRF_SECURITY_ADVANCED
-    default y
+    default y if !CHIP_CRYPTO_PSA
```

Comment:
nit: would it work to have `if !CHIP_CRYPTO_PSA` in line 294 for all mbedtls related config?